cdec/installation-manager-resources/src/main/resources/im-scripts/install-codenvy.sh

```diff
@@ -914,11 +920,18 @@ doCheckAvailableResourcesLocally() {
     local availableDiskSpaceToDisplay=$(( availableDiskSpace /1000/1000 ))
     local availableDiskSpaceToDisplay=$(printf "%-11s" "${availableDiskSpaceToDisplay} GB")
 
+    local pathToCheckWriteAccess=$([[ -n $INSTALL_DIRECTORY ]] && echo $INSTALL_DIRECTORY || pwd)
+    if (( $(touch "$pathToCheckWriteAccess/tmp_file" &>/dev/null; echo $?; rm "$pathToCheckWriteAccess/tmp_file" &>/dev/null) == 1 )); then
+        writeAccessIssueFound=true
+        writeAccessStateToDisplay=$(printf "%-44s" && printError "[NOT OK]")
+    fi
+
     println
     println "                MINIMUM        RECOMMENDED     AVAILABLE"
     println "RAM             $minRAMToDisplay $recRAMToDisplay $availableRAMToDisplay $RAMStateToDisplay"
     println "CPU             $minCoresToDisplay $recCoresToDisplay $availableCoresToDisplay $coresStateToDisplay"
     println "Disk Space      $minDiskSpaceToDisplay $recDiskSpaceToDisplay $availableDiskSpaceToDisplay $diskStateToDisplay"
+    println "Write Access    $writeAccessStateToDisplay"
```

Comment:
If you use "Write Permission" then you will have to adjust the columns as the formatting will be altered.